x/ibc/testing/solomachine.go

```diff
@@ -90,19 +98,35 @@ func (solo *Solomachine) CreateHeader() *solomachinetypes.Header {
 
 // CreateMisbehaviour constructs testing misbehaviour for the solo machine client
 // by signing over two different data bytes at the same sequence.
-func (solo *Solomachine) CreateMisbehaviour() *solomachinetypes.Misbehaviour {
+func (solo *Solomachine) CreateMisbehaviour(cdc codec.BinaryMarshaler) *solomachinetypes.Misbehaviour {
 	dataOne := []byte("DATA ONE")
 	dataTwo := []byte("DATA TWO")
 
-	sig, err := solo.PrivateKey.Sign(append(sdk.Uint64ToBigEndian(solo.Sequence), dataOne...))
+	signBytes := &solomachinetypes.SignBytes{
```

Comment:
not sure if this should include timestamp

Reply:
we should and the diversifier, see my other response